### PR TITLE
feat: add ses email info doc and few FAQs

### PR DESCRIPTION
**Pull request by jainpawan21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Mailgun](https://github.com/novuhq/novu/tree/main/providers/mailgun)
 - [x] [SES](https://github.com/novuhq/novu/tree/main/providers/ses)
 - [x] [Postmark](https://github.com/novuhq/novu/tree/main/providers/postmark)
-- [x] [NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
+- [x] [Custom SMTP / NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```
Comment (scopsy) [inline README.md:130]:
```suggestion
- [x] [Custom SMTP](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```

File: docs/docs/community/monorepo-structure.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [Mailgun](https://github.com/novuhq/novu/tree/main/providers/mailgun)
 - [SES](https://github.com/novuhq/novu/tree/main/providers/ses)
 - [Postmark](https://github.com/novuhq/novu/tree/main/providers/postmark)
-- [NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
+- [Custom SMTP / NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```
Comment (scopsy) [inline docs/docs/community/monorepo-structure.md:98]:
```suggestion
- [Custom SMTP](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```

File: packages/node/README.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Mailgun](https://github.com/novuhq/novu/tree/main/providers/mailgun)
 - [x] [SES](https://github.com/novuhq/novu/tree/main/providers/ses)
 - [x] [Postmark](https://github.com/novuhq/novu/tree/main/providers/postmark)
-- [x] [NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
+- [x] [Custom SMTP / NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```
Comment (scopsy) [inline packages/node/README.md:90]:
```suggestion
- [x] [Custom SMTP](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```

File: packages/stateless/README.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Mailgun](https://github.com/novuhq/novu/tree/main/providers/mailgun)
 - [x] [SES](https://github.com/novuhq/novu/tree/main/providers/ses)
 - [x] [Postmark](https://github.com/novuhq/novu/tree/main/providers/postmark)
-- [x] [NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
+- [x] [Custom SMTP / NodeMailer](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```
Comment (scopsy) [inline packages/stateless/README.md:67]:
```suggestion
- [x] [Custom SMTP](https://github.com/novuhq/novu/tree/main/providers/nodemailer)
```

File: docs/docs/channels/email/ses.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- Either use root aws account or create a new IAM account with appropriate permission policies.
+- Create new access key and save generated `ACCESS_KEY_ID` and `ACCESS_SECRET_KEY` carefully
+- Choose Amazon Simple Email Service.
+- Create new identity.
+- Either choose domain or email.
+- Verify your domain (by addding few DNS records as mentioned in SES instructions) or email (AWS sends a verification email to your email).
+- Verify recipient email also by creating new email identity type [only in sandbox mode].
+- Test if your SES is setup correctly using test email feature.
+
+## Create a SendGrid integration with Novu
```
Comment (scopsy) [inline docs/docs/channels/email/ses.md:20]:
```suggestion
## Create a SES integration with Novu
```

File: docs/docs/channels/email/ses.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Amazon SES
+
+You can use the [Amazon SES](https://aws.amazon.com/ses/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
+
+## Getting Started
+
+To use the Amazon SES provider, you will need to create a Sendgrid account and add your credentials to the Amazon SES integration on the Novu platform.
```
Comment (scopsy) [inline docs/docs/channels/email/ses.md:7]:
```suggestion
To use the Amazon SES provider, you will need to create a SES account and add your credentials to the Amazon SES integration on the Novu platform.
```

**Review by p-fernandez — APPROVED** [review]
Documentation improvement is always a great PR. Thanks for this.
🌟

**Comment by jainpawan21** [comment]
Can we please merge this PR?

**Pull request merged**